### DISCUSSION TITLE
Stock reservation in checkout

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1016] [Feature Requests]
When a customer goes to checkout out, the system should ensure there is suitable stock available and also reserve that stock for the checkout session period. If the customer does not complete the checkout in the session time, then the stock should be automatically unreserved and available to purchase again.

**Comment by SrPatinhas** [comment]
Would be possible to have this option on the settings of the store? And should this be a toggle on/off/for how long is reserved? or will it always have a default value and we need to change it in the some place in the code/files?
